common/instructions.coffee

```diff
@@ -87,8 +76,8 @@ module.exports =
 		Unmount your USB key with <code>umount /dev/sddN</code> where N is your USB key number.
 	'''
 
-	EJECT_SD: 'Safely eject the freshly burnt SD card and insert into the <%= TYPE_NAME %>.'
-	EJECT_USB: 'Safely eject the freshly burnt USB key and insert into the <%= TYPE_NAME %>.'
+	EJECT_SD: 'Insert the freshly burnt SD card into the <%= TYPE_NAME %>.'
+	EJECT_USB: 'Insert the freshly burnt USB key into the <%= TYPE_NAME %>.'
```

Comment:
Based declared name, it seems weird that we don't mention `eject` any more, but it's totally more straightforward :+1: